main.js

```diff
@@ -223,8 +231,9 @@ app.whenReady().then(() => {
 
   // Proxy user actions on buttons between immersive browser window
   // and the main offscreen content window
-  ipcMain.handle('toggle3d', _ => {
-    contentWindows[0].webContents.send('toggle3d');
+  ipcMain.handle('toggle3d', async () => {
+    await contentWindows[0].loadFile("content.html");
```

Comment:
I'm planning to rework this so that we end up in a situation where the "3d mode" does not mean "loading content.html" but rather "enable 3d mode on the currently loaded page".